static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -63,6 +66,7 @@ import ThresholdsChart from './thresholdsChart';
 
 type Props = {
   aggregate: MetricRule['aggregate'];
+  anomalies: Anomaly[];
```

Comment:
i'd kinda expect these to be loaded as something else into the Chart component (rather than anomalies). something more like a generic for data or something along those lines? 🤔